tests/test_cli_extended.py

```
@@ -0,0 +1,396 @@
+
+import pytest
+
+from fetchtastic import cli
+
+
+@pytest.mark.user_interface
+@pytest.mark.unit
+class TestCLIHelpFunctionality:
+    """Test CLI help functionality."""
+
+    def test_show_help_basic(self, mocker):
```

Ruff (0.14.2)

Unused method argument: `mocker`

(ARG002)